frontend/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
where are we using environment variables?

Reply:
This adds typing to vite's environment api (https://vitejs.dev/guide/env-and-mode)